packages/wdio-allure-reporter/tests/cucumber.suite.test.ts

```diff
@@ -57,80 +62,79 @@ describe('reporter option "useCucumberStepReporter" set to true', () => {
                 reporter.onHookStart(cucumberHelper.hookStart())
                 reporter.addAttachment(attachmentHelper.xmlAttachment())
                 reporter.onHookEnd(cucumberHelper.hookEnd())
+
                 const suiteResults: any = { tests: [cucumberHelper.testPass()], hooks: new Array(2).fill(cucumberHelper.hookEnd()) }
+
                 reporter.onSuiteEnd(cucumberHelper.scenarioEnd(suiteResults))
                 reporter.onSuiteEnd(cucumberHelper.featureEnd(suiteResults))
                 reporter.onRunnerEnd(runnerEnd())
 
-                const results = getResults(outputDir)
-                expect(results).toHaveLength(2) // one for report, one for attachment
-                allureXml = results.find((xml: any) => xml('ns2\\:test-suite').length >= 1)
+                const { results, attachments, containers } = getResults(outputDir)
+
+                allureResult = results[0]
```

Comment:
Please assert the length before accessing the value. I don't want to spam you with this comment, but please take a look and fix it, whatever it may be.

Reply:
:+1: